src/components/inputs/inputMap.js

```diff
@@ -32,7 +31,6 @@ const inputMap = {
   [fieldTypes.date]: DateInput,
   [fieldTypes.daterange]: DateRangeInput,
   [fieldTypes.feetmeters]: FeetMetersInput,
-  [fieldTypes.relationships]: RelationshipsInput,
```

Comment:
Can the `RelationshipsInput` file also be deleted?

Reply:
yes